lib/plugin_version.rb

```diff
@@ -80,10 +80,26 @@ def check_manifest_version_validity
   def check_targets_validity
```

Comment:
I don't think this name fits, this method is setting targets, not checking validity.